portfolio/src/main/webapp/script.js

```diff
@@ -15,33 +15,32 @@
 /**
  * Adds a random fact to the page.
  */
-function addRandomFact() {
+function addRandomFact () {
   const facts =
-    ['I am from Donegal in Ireland', 
-    'I can speak English, Irish, French and a little Swedish', 
-    'I love archery, and I am a member of my college\'s archery club', 
-    'My favourite video game is The Legend of Zelda: Breath of the Wild',
-    'My favourite TV show is Erased',
-    'I have two sisters, one older and one younger'
-    ];
+    ['I am from Donegal in Ireland',
+      'I can speak English, Irish, French and a little Swedish',
+      'I love archery, and I am a member of my college\'s archery club',
+      'My favourite video game is The Legend of Zelda: Breath of the Wild',
+      'My favourite TV show is Erased',
+      'I have two sisters, one older and one younger'
+    ]
 
   // Pick a random fact.
-  const fact = facts[Math.floor(Math.random() * facts.length)];
+  const fact = facts[Math.floor(Math.random() * facts.length)]
```

Comment:
Don't rely on automatic ; insertion for JS: it is super-complicated https://stackoverflow.com/questions/2846283/what-are-the-rules-for-javascripts-automatic-semicolon-insertion-asi

Reply:
Thanks, I didn't know about this!